controller/controller_test.go

```diff
@@ -32,7 +32,7 @@ func TestHandle(t *testing.T) {
 	_ = os.Setenv("ROUTER_TEST", "true")
 	_ = os.Setenv("ENVIRONMENT", "dev")
 	defer os.Clearenv()
-	Handle(cfg.Security, cfg.Web, cfg.UI, cfg.Metrics)
+	Handle(cfg.Security, cfg.Web, cfg.UI, cfg, cfg.Metrics)
```

Comment:
Assuming the answer to my previous comment is `no`, this change would be unnecessary.
If not, then we'd just need to pass `cfg`, no need to pass `cfg.Security`,`cfg.Web`, `cfg.UI` and `cfg.Metrics` since they're all under `cfg` anyways.

Reply:
Yes this definitely needs to be refactored

Reply:
Since we want to set the endpoints dedicated, we thought it would be better placed under the `endpoints.UIConfig`
I refactored the code a bit and reduced the config parameters to the minimum.